public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -9,98 +9,105 @@ import { DetailsStep } from './DetailsStep';
 import { QueryStep } from './QueryStep';
 import { useForm, FormContext } from 'react-hook-form';
 
-import { GrafanaAlertState } from 'app/types/unified-alerting-dto';
 //import { locationService } from '@grafana/runtime';
```

Comment:
nit: could probably be cleaned up since we are doing changes here.

Reply:
- [ ] clean up